### PR TITLE
manual update for dependencies after security alert

### PR DESCRIPTION
**Pull request by vhvb1989**
Using override for minimatch version

### CODE REVIEW

File: templates/todo/web/react-fluent/package-lock.json [hunk 1]
```diff
@@ -13296,7 +13296,7 @@
       "integrity": "sha512-nRCcW9Sj7NuZwa2XvH9co8NPeXUBhZP7CRKJtU+cS6PW9FpCIFoI5ib0NT1ZrbNuPoRy0ylyCaUL8Gih4LSyFg==",
       "dev": true,
       "dependencies": {
-        "minimatch": "3.0.4"
+        "minimatch": "3.0.5"
```
Comment (rajeshkamal5050) [inline templates/todo/web/react-fluent/package-lock.json:13299]: Similar changes not required for generators/repo/package-lock.json?

Comment (vhvb1989) [reply templates/todo/web/react-fluent/package-lock.json:13299]: In all other places, the dependecy is listed as `^3.0.4`, which makes `npm` to automatically look for the latest version.
There are only these 2 cases where the version is pinned directly.

Comment (wbreza) [reply templates/todo/web/react-fluent/package-lock.json:13299]: True - but the package-lock.json will force it to use the version stamped in it at time of last commit vs pulling latest version.

**Review by wbreza — APPROVED** [review]
Please verify if auto upgrade happens or not when performing `npm install` for the repoman project.  Otherwise looks good.

File: templates/todo/web/react-fluent/package.json [hunk 2]
```diff
@@ -65,6 +65,13 @@
     "async": "^3.2.3",
     "nth-check": "^2.0.1",
     "ejs": "^3.1.7",
-    "svgo": "^2.8.0"
+    "svgo": "^2.8.0",
+    "react-scripts":{
+      "react-dev-utils": {
+        "recursive-readdir": {
+          "minimatch": "3.0.5"
```
Comment (ellismg) [inline templates/todo/web/react-fluent/package.json:72]: Should we change this to `^3.0.5` so if there's another security update we can just regenerate the lock file?

**Review by ellismg — APPROVED** [review]
One question about if we should be using a `^` constraint here or not.  If you do change it, I /think/ you'll need to run `npm install` once more to regenerate the lock file.

File: templates/todo/web/react-fluent/package.json (hunk 2 above)
Comment (vhvb1989) [reply templates/todo/web/react-fluent/package.json]: Done

**Pull request merged**